Alpaca.Markets/RestClient.Polygon.cs

```diff
@@ -160,6 +160,70 @@ public Task<IEnumerable<IExchange>> ListExchangesAsync()
                 _polygonHttpClient, FakeThrottler.Instance, builder);
         }
 
+        /// <summary>
+        /// Gets list of historical minute bars for single asset from Polygon REST API endpoint.
+        /// </summary>
+        /// <param name="symbol">>Asset name for data retrieval.</param>
+        /// <param name="multiplier">>Number of bars to combine in each result.</param>
+        /// <param name="dateFromInclusive">Start time for filtering (inclusive).</param>
+        /// <param name="dateToInclusive">End time for filtering (inclusive).</param>
+        /// <param name="unadjusted">Set to true if the results should not be adjusted for splits.</param>
+        /// <returns>Read-only list of minute bars for specified asset.</returns>
+        public Task<IHistoricalItems<IAgg>> ListMinuteAggregatesAsyncV2(
```

Comment:
Remove `V2` suffix from both new methods - it's just an implementation detail which should be visible to SDK clients. These methods will have same names as existing and it will make future transitions for clients more simple.